File: JAVA/Clase5-Varios/Poo.java
class Person
{
    public String name;
    public int age;

    public Person()
    {
        name = "";
        age = 0;
    }

    public void sayHello()
    {
        System.out.println("Hi, I'm " + name + " and I'm " + age + " years old.");
    }
}

public class Poo
{
    public static void main(String[] args)
    {
        Person p1 = new Person();
        p1.name = "Adrian";
        p1.age = 20;

        p1.sayHello();
    }
}

File: C++/Clase5-Varios/poo.cpp
#include <iostream>
using namespace std;

class Person
{
public:
    string name;
    int age;

    Person()
    {
        name = "";
        age = 0;
    }

    void say_hello()
    {
        cout << "Hi, I'm " << name << " and I'm " << age << " years old.\n";
    }
};

int main()
{
    Person p1;
    p1.name = "Adrian";
    p1.age = 20;

    p1.say_hello();
    return 0;
}